charts/incubator/flashpaper/values.yaml

```diff
@@ -0,0 +1,51 @@
+image:
+  pullPolicy: IfNotPresent
+  repository: ghcr.io/andrewpaglusch/flashpaper
+  tag: latest@sha256:91e23442eec448d0ceea4e71a277fb204d22d5ed0980570782403aa7c0e8400e
+persistence:
+  data:
+    enabled: true
+    mountPath: /var/www/html/data
+  varrun:
+    enabled: false
+
+portal:
+  open:
+    enabled: true
+securityContext:
+  container:
+    readOnlyRootFilesystem: false
+    runAsGroup: 0
+    runAsNonRoot: false
+    runAsUser: 0
+service:
+  main:
+    ports:
+      main:
+        port: 8544
+        protocol: tcp
+        targetPort: 80
+workload:
+  main:
+    podSpec:
+      containers:
+        main:
+          env:
+            ANNOUNCEMENT: ""
+            MAX_SECRET_LENGTH: "3000"
```

Comment:
Why are here set as string and on the quests as int?
Can you please sync those formats?

Reply:
This still needs to be resolved, can cause scale/ui issues down the line